app/lib/efile/ny/nys_tax_computation.rb

```diff
@@ -3,132 +3,99 @@ module Ny
     class NysTaxComputation
       # Implements the NYS Tax Computation, for use with NY AGI over $107,650, as documented in the IT-201 instructions
       # https://www.tax.ny.gov/forms/html-instructions/2023/it/it201i-2023.htm#tax-computation
-      # Worksheet numbers in the comments correspond to worksheets in the instructions
-
-      AGI_TOP_BRACKET_THRESHOLD = 25_000_000 # used as the upper AGI limit for all mid and bottom bracket worksheets
+      # Worksheet numbers in the comments correspond to schedules in the instructions
 
       class << self
         def calculate(agi, taxable_income, filing_status)
-          # Get the set of worksheets for the filing status
-          worksheets = worksheets_for_filing_status(filing_status)
-          # Find the correct worksheet based on AGI and taxable income
-          worksheet = worksheets.find { |w| w.valid_for(agi, taxable_income) }
-          # use the worksheet to compute the NYS tax amount
-          worksheet.compute(agi, taxable_income, filing_status)
+          # https://www.tax.ny.gov/forms/html-instructions/2023/it/it201i-2023.htm#tax-computation
+          line1 = agi
+          line2 = taxable_income
+          line3 = line2 * 0.055
```

Comment:
What do line 1 and 2 correspond to here? I think just keeping agi and taxable_income would make it more readable

Reply:
Are they the 201 lines?